src/layout/Nav.tsx

```diff
@@ -1,23 +1,67 @@
-import { Typography, AppBar, Toolbar, Box } from '@mui/material';
+import { useState, MouseEvent } from 'react';
+import { Typography, AppBar, Toolbar, Box, useMediaQuery, IconButton, Menu, MenuItem } from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
 import logo from '../pics/logo.png';
 import NavButton from '../components/NavButton';
+import { NavLink } from 'react-router-dom';
 
 function Nav() {
+  const isMobile = useMediaQuery('(max-width:600px)');
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
   return (
-    <AppBar className='py-1.5' component='div' position='sticky' >
-      <Toolbar component="div">
-        {/* logo and title */}
-        <Box className='h-16 w-16 mr-2' component='img' src={logo} alt='book' />
-        <Typography className='grow' component='div' variant='h4' color='secondary'>Hikma AI</Typography>
+    // flex sm:justify-between sm:flex-row flex-col text-center
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar className='py-1.5' component='div' position='sticky' >
+        <Toolbar component="div">
+          {/* logo and title */}
+          <Box className='h-16 w-16 mr-2' component='img' src={logo} alt='book' />
+          <Typography className='grow' component='div' variant='h4' color='secondary'>Hikma AI</Typography>
 
-        {/* nav links */}
-        <Box>
-          <NavButton to='/'>Home</NavButton>
-          <NavButton to='/about'>About</NavButton>
-          <NavButton to='/add'>Add</NavButton>
-        </Box>
-      </Toolbar>
-    </AppBar>
+          {/* nav links */}
+          {!isMobile
```

Comment:
Nit: This function is getting a little to big maybe you can mobile to Links to a smaller function in the same code. Something like MenuItems or something

Reply:
You mean this whole thing in a its own function (same file)?
```
          {!isMobile
            ? <Box>
                <NavButton to='/'>Home</NavButton>
                <NavButton to='/about'>About</NavButton>
                <NavButton to='/add'>Add</NavButton>
              </Box>
            : <IconButton
                size="large"
                edge="start"
                color="secondary"
                aria-label="menu"
                id="basic-button"
                aria-controls={open ? 'basic-menu' : undefined}
                aria-haspopup="true"
                aria-expanded={open ? 'true' : undefined}
                onClick={handleClick}
              >
                <MenuIcon />
              </IconButton>
          }
```